src/components/Button/Button.js

```diff
@@ -54,7 +55,7 @@ class Button extends Component<ButtonProps> {
         type={ type }
         loading={ loading }
         onClick={ this.onClick }
-        aria-busy={ String(loading) }
+        aria-busy={ String(!!loading) }
```

Comment:
@ej9x for what? We already have a boolean prop

Reply:
Maybe `Boolean(loading).toString()`?

Reply:
I think it's more reasonable and clear.

Reply:
Saw this. What does 'aria' word mean?

Reply:
@andrei-anisimov attribute for detecting loading state of the button

Reply:
'aria' - strange word. Did you mean 'area'?

Reply:
@andrei-anisimov aria stands for "Accessible Rich Internet Applications". aria-* tags are used for accessibility.

Reply:
@vorobeez got you, thanks for the explanation!